### ISSUE TITLE
[Gallery Submission Feedback] Adding https://github.com/targito/TargitoTrackingTemplate repository to the template gallery #1722

### ISSUE THREAD

**Issue by julianhsu8** [targito/TargitoTrackingTemplate#1]
Your documentation link is invalid, and the template appears to be asking the user to populate the trackingDomain.  Is this latter intended?

**Comment by jauzah** [comment]
Yes, both intended and correct. Documentation is client-specific and client needs to log in to Targito. We also use 1st party tracking and therefore the correct trackingDomain must be specified.
